hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -40,6 +40,11 @@
     private String fullPath;
 
     /**
+     * Relative path to the file from the base path
+     */
+    private String relativePath;
```

Comment:
yeah. fullPath can be removed from the `fileIdAndFullPaths` section of the commit metadata..